src/main/java/at/hannibal2/skyhanni/utils/ItemUtils.kt

```diff
@@ -213,7 +225,9 @@ object ItemUtils {
                     "internal name" to getInternalName(),
                     "item name" to name,
                     "inventory name" to InventoryUtils.openInventoryName(),
+                    "pattern result" to rarity,
```

Comment:
why pattern result? isnt rarity as label here better?
also where is the beta only tag for item rarity stack error log

Reply:
Since it is the result of the pattern. And yes I forgot to add betaOnly here but added this already